docs/combined_model_dag.md

```diff
@@ -0,0 +1,145 @@
+
+```
+
+### OVMS configuration
+
+```json
+{
+    "model_config_list": [
+        {
+            "config": {
+                "name": "emotions-recognition",
+                "base_path": "/models/emotions-recognition-retail-0003"
+            }
+        },
+        {
+            "config": {
+                "name": "age-gender-recognition",
+                "base_path": "/models/age-gender-recognition-retail-0013"
+            }
+        }
+    ],
+    "pipeline_config_list": [
+        {
+            "name": "combined-recognition",
+            "inputs": ["image"],
+            "nodes": [
+                {
+                    "name": "emotions",
+                    "model_name": "emotions-recognition",
+                    "type": "DL model",
+                    "inputs": [
+                        {"data": {"node_name": "request",
+                                   "data_item": "image"}}
+                    ], 
+                    "outputs": [
+                        {"data_item": "prob_emotion",
+                         "alias": "emotion"}
+                    ] 
+                },
+                {
+                    "name": "age-gender",
+                    "model_name": "age-gender-recognition",
+                    "type": "DL model",
+                    "inputs": [
+                        {"data": {"node_name": "request",
+                                  "data_item": "image"}}
+                    ], 
+                    "outputs": [
+                        {"data_item": "age_conv3",
+                         "alias": "age"},
+                        {"data_item": "prob",
+                         "alias": "gender"}
+                    ] 
+                }
+            ],
+            "outputs": [
+                {"age": {"node_name": "age-gender",
+                           "data_item": "age"}},
+                {"gender": {"node_name": "age-gender",
+                           "data_item": "gender"}},
+                {"emotion": {"node_name": "emotions",
+                           "data_item": "emotion"}}
+            ]
+        }
+    ]
+}
+```
+
+### Starting docker container
+```bash
+docker run -d -p 9000:9000 -v ${PWD}/models:/models -v ${PWD}/config.json:/config.json openvino/model_server --config_path /config.json --port 9000
+```
+
+### Using the client 
+
+Get the sample image:
+```
+wget https://github.com/openvinotoolkit/open_model_zoo/raw/master/models/intel/age-gender-recognition-retail-0013/description/age-gender-recognition-retail-0001.jpg
+```
+
+Use the following code snippet to submit a request via gRPC to the serving and display the response:
+
+```python
```

Comment:
Maybe we should add this code to example_clients as example client for pipeline request?